src/app/core/admin/admin-entity-details/admin-entity-details/admin-entity-details.component.html

```diff
@@ -33,8 +33,7 @@
           [config]="componentConfig.config"
           (configChange)="componentConfig.config = $event"
           [entityType]="entityConstructor"
-          [tabIndex]="tabIndex"
-          [sectionIndex]="i"
+          [uniqueAreaId]="tabIndex + i"
```

Comment:
this can create overlap (e.g. tabIndex 11 and sectionIndex 1 === tabIndex 1 and sectionIndex 11). Fixed this with a separation character